frontend/src/config/kyrgyzstan/prism.json

```diff
@@ -16,13 +17,12 @@
   },
   "alertFormActive": false,
   "map": {
-    "boundingBox": [69.46, 39.28, 80.26, 43.3]
+    "boundingBox": [68, 39.28, 80.26, 43.3]
   },
   "defaultDisplayBoundaries": ["admin_boundaries", "admin1_boundaries"],
   "serversUrls": {
     "wms": [
-      "https://api.earthobservation.vam.wfp.org/ows/wms",
-      "https://kyrgyzstan.sibelius-datacube.org:5000/wms"
```

Comment:
@wadhwamatic do you want to merge this or is it just to prevent the app from hanging?

Reply:
it should be merged. i've made this a working branch for updates while i'm here in bishkek